src/libstd/macros.rs

```diff
@@ -33,6 +33,8 @@
 /// panic!(4); // panic with the value of 4 to be collected elsewhere
 /// panic!("this is a {} {message}", "fancy", message = "message");
 /// ```
+///
+/// [`Box<Any>`]: ../../collections/boxed/struct.Box.html
```

Comment:
Good catches, thanks. My web server was eating the extra '../' so my dead link checker didn't catch that.

Is there any situation where a type is mirrored into `std` that I shouldn't treat the `std` version as canonical? e.g. if something in collections mentions Box is it ok for that link to be to `std`, or will I need custom resolution per lib?